packages/webapp-libs/webapp-contentful/src/routes/demoItems/demoItemListItem/demoItemListItem.component.tsx

```diff
@@ -21,26 +20,45 @@ export const DemoItemListItem = ({ id, item }: DemoItemListItemProps) => {
   const generateLocalePath = useGenerateLocalePath();
 
   return (
-    <Container>
-      <Link to={generateLocalePath(RoutesConfig.demoItem, { id })}>
-        <FavoriteIcon
-          role="checkbox"
-          aria-checked={isFavorite}
-          aria-label={intl.formatMessage({
-            defaultMessage: 'Is favorite',
-            id: 'Demo Item / Is favorite',
-          })}
-          onClick={(e) => {
-            e.preventDefault();
-            setFavorite(!isFavorite);
-          }}
-        >
-          <Icon icon={isFavorite ? favoriteIconFilled : favoriteIconOutlined} />
-        </FavoriteIcon>
-
-        {item.image && <Thumbnail {...imageProps(item.image, { size: { height: 50 } })} role="presentation" />}
-        <Title>{item.title}</Title>
+    <li className="flex items-center w-[100%]">
+      <Link
+        className="p-8 w-[100%] justify-start hover:no-underline"
+        to={generateLocalePath(RoutesConfig.demoItem, { id })}
+      >
+        {item.image && (
+          <img
+            className="object-cover rounded w-16 h-12"
+            {...imageProps(item.image, { size: { height: 50 } })}
+            role="presentation"
+          />
+        )}
+        <p className="px-4">{item.title}</p>
       </Link>
-    </Container>
+
+      <Button
+        variant="ghost"
+        role="checkbox"
+        aria-checked={isFavorite}
+        aria-label={intl.formatMessage({
+          defaultMessage: 'Is favorite',
+          id: 'Demo Item / Is favorite',
+        })}
+        className="mr-4"
+        onClick={(e) => {
+          setFavorite(!isFavorite);
+        }}
+      >
+        <Star
+          className={cn({
```

Comment:
I think we can simplify this one
`
className={cn({
  'fill-none text-slate-700 dark:fill-none': !isFavorite,
  'fill-slate-700 dark:fill-slate-400 dark:text-slate-400': isFavorite,
})}
`